pkg/scheduler/scheduler_test.go

```diff
@@ -1284,6 +1386,100 @@ func TestEntryOrdering(t *testing.T) {
 	}
 }
 
+func TestClusterQueueUpdate(t *testing.T) {
```

Comment:
+1 on this test, but for a separate PR, as it shouldn't be necessary for the failure scenario described.

Reply:
nvm